xicam/gui/bluesky/search.py

```diff
@@ -0,0 +1,430 @@
+"""
+Experimental Qt-based data browser for bluesky
+"""
+import ast
+from datetime import datetime
+import event_model
+import functools
+import itertools
+import jsonschema
+import logging
+import queue
+import threading
+import time
+
+from qtpy.QtCore import Qt, Signal, QThread
+from qtpy.QtGui import QStandardItemModel, QStandardItem
+from qtpy.QtWidgets import (
+    QAbstractItemView,
+    QPushButton,
+    QComboBox,
+    QDateTimeEdit,
+    QHeaderView,
+    QHBoxLayout,
+    QMessageBox,
+    QLabel,
+    QLineEdit,
+    QVBoxLayout,
+    QWidget,
+    QTableView,
+    )
+from .utils import ConfigurableQObject
+from .top_utils import load_config, Callable
+
+
+MAX_SEARCH_RESULTS = 100  # TODO Use fetchMore instead of a hard limit.
+log = logging.getLogger('bluesky_browser')
```

Comment:
This should probably be changed to `'search_widget'` or, if Xi-cam already has a convention for logging scope, `'SOMETHING.search_widget'`.